oShare/Peer Browser/PeerBrowserTableViewHandler.swift

```diff
@@ -9,7 +9,7 @@ class PeerBrowserTableViewHandler: NSObject, UITableViewDelegate, UITableViewDat
 	weak var appDelegate: AppDelegate?
 	
 	// Keep track of the number of peers. Since this is strictly for data display, we don't want to access it outside of this handler.
-	private var foundPeers: [MCPeerID] = []
+	private(set) var foundPeers: [MCPeerID] = []
```

Comment:
This is to enable direct testing on this property, without needing to write a fake/mock.